packages/editor/src/hooks/useMediaLibraryModal.ts

```diff
@@ -201,10 +200,33 @@ export const useMediaLibraryModal = (editor: Editor | null) => {
                 // No text is currently selected.
                 // => Insert the name of the link and select it.
                 insertAndSelectText(text);
+              } else {
+                const selection = editor.view.state.selection;
+                if (
+                  text &&
+                  editor.view.state.selection.content().content.child(0)
+                    .textContent !== text
+                ) {
+                  editor
+                    .chain()
+                    .focus()
+                    .insertContentAt(
+                      {
```

Comment:
```suggestion
                      selection
```

ou 
`const {from, to} = selection;` et réutiliser les 2 variables ensuite